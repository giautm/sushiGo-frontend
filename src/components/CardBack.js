import React from 'react';

const CardBack = props => {
  return <div>CardBack</div>;
};

export default CardBack;
