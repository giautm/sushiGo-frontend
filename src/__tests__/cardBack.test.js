import React from 'react';
import CardBack from '../components/CardBack';
import renderer from 'react-test-renderer';

const fakeRollData = {
  id: '1',
  imageNumber: '8',
  name: 'Spicy Tuna Roll',
  ingredients: 'Tuna, Chili sauce, Nori, Rice'
};

describe('CardBack', () => {
  it('renders correctly', () => {
    const component = renderer.create(<CardBack rollData={fakeRollData} />);
    expect(component).toMatchSnapshot();
  });
});
