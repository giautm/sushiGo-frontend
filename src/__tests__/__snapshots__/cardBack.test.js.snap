// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBack renders correctly 1`] = `
<div
  className="card back"
>
  <ul>
    <li>
      Tuna
    </li>
    <li>
      Chili sauce
    </li>
    <li>
      Nori
    </li>
    <li>
      Rice
    </li>
  </ul>
  <div
    className="line"
  />
</div>
`;
